index.html

```diff
@@ -0,0 +1,103 @@
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./css/product.css">
+</head>
+
+<body>
+    <header>
+        <div class="nav1-container">
+            <nav class="nav1">
+                <li id="Products" class="active">
+                    <a href="./index.html">Products</a>
+                    <span></span>
+                </li>
+                <li id="Cart">
+                    <a href="./html/cart.html">Cart</a>
+                    <span></span>
+                </li>
+                <li id="Seller">
+                    <a href="./html/seller.html">Seller</a>
+                    <span></span>
+                </li>
+
+                <li class="dropdownli" onclick="clicked1(this)">
+                    <div class="dropdown">
+                        <button class="dropbtn">Category <i class="down-arrow"></i></button>
+                        <div id="dropdown-items">
+                            <a href="#" onclick="dropItem1(this)">Music</a>
+                            <a href="#" onclick="dropItem1(this)">TV</a>
+                            <a href="#" onclick="dropItem1(this)">Weather</a>
+                            <a href="#" onclick="dropItem1(this)">Sounds</a>
+                        </div>
+                    </div>
+                    <span></span>
+                </li>
+                <li class="dropdownli1" onclick="clicked2(this)">
+                    <div class="dropdown1">
+                        <button class="dropbtn1">Price <i class="down-arrow"></i></button>
+                        <div id="dropdown-items1">
+                            <a href="#" onclick="dropItem2(this)">Music</a>
+                            <a href="#" onclick="dropItem2(this)">TV</a>
+                            <a href="#" onclick="dropItem2(this)">Weather</a>
+                            <a href="#" onclick="dropItem2(this)">Sounds</a>
+                        </div>
+                    </div>
+                    <span></span>
+                </li>
+                <li id="search"><input type="search" name="" id="" placeholder="Search..."><svg viewBox="0 0 32 32" width="1em" height="1em" class="ssrcss-xi5oyi-StyledIcon e6m7o991" focusable="false" aria-hidden="true"><path d="M13 24.3c6.5 0 11.7-5.1 11.7-11.7S19.6 1 13 1 1.4 6.1 1.4 12.7 6.5 24.3 13 24.3zm0-2.6c-5.1 0-9-3.9-9-9.1s3.9-9.1 9-9.1 9 3.9 9 9.1-3.9 9.1-9 9.1zm5.8.4l8.9 8.9 2.9-2.9-8.9-8.9-2.9 2.9z"></path></svg></li>
+
+            </nav>
+            <li class="icon1" onclick="toggle1()">
+                <div class="icon1-container" onclick="myFunction1(this)">
+                    <div class="bar1-1"></div>
+                    <div class="bar2-1"></div>
+                    <div class="bar3-1"></div>
+                </div>
+            </li>
+        </div>
+
+    </header>
+    <script>
```

Comment:
Why repeating yourself!
and don't put script here, we have our files for scripts